src/agent/src/config.rs

```diff
@@ -27,6 +27,7 @@ const CONFIG_FILE: &str = "agent.config_file";
 const CONTAINER_POLICY_FILE: &str = "agent.container_policy_file";
 const HTTPS_PROXY: &str = "agent.https_proxy";
 const NO_PROXY: &str = "agent.no_proxy";
+const ENABLE_DATA_INTEGRITY: &str = "agent.enable_data_integrity";
```

Comment:
This is fine, but thinking to the future, do you think there is any chance we might want to pass additional values here? If so, maybe something like `agent.data_integrity=[true|false]` would be better for now (so we could enhance that "language" later ( `agent.data_integrity=[true|false|foo=x,bar=y,baz=(1,2,"hello,world")]`).